arcsi/api/utils.py

```diff
@@ -6,7 +6,7 @@
 from arcsi.model.item import Item
 from flask import current_app as app
 from slugify import slugify
-from werkzeug import secure_filename
+from werkzeug.utils import secure_filename
```

Comment:
You got me with this one, as I remember I tried out a newer version of flask-security which needed newer werkzeug as its dependency, and it caused ImportError in that case.
Since we won't do any version update here, maybe it is better just to revert this change, and do it when it is necessary.